app/services/creators/capital_eligibility_creator.rb

```diff
@@ -18,22 +18,35 @@ def call
     def create_eligibility(ptc)
       return if eligibility_record_exists?(ptc)
 
-      @summary.eligibilities.create!(
-        proceeding_type_code: ptc,
-        upper_threshold: upper_threshold(ptc),
-        lower_threshold:,
-        assessment_result: "pending",
-      )
+      if @assessment.level_of_representation == "controlled"
+        @summary.eligibilities.create!(
+          proceeding_type_code: ptc,
+          upper_threshold: controlled_threshold(ptc),
```

Comment:
I had been trying to prevent the code from knowing that for controlled work the upper and lower thresholds were equal. But I guess it's highly unlikely this will ever change without other things _also_ changing, so hoping to be able to contain such a change purely in config is probably unrealistic.